.build/cassandra-build-deps-template.xml

```diff
@@ -123,5 +123,9 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock-jre8</artifactId>
```

Comment:
should whether it is `jre8` or `jre11` depend on which Java version we compile it against?

Reply:
I run the tests with Java 11 and it just works. I think this is backward compatible. `jre11` probably can not be run with Java 8 but jre8 can be run with Java 11 just fine.